src/connect/components/breadcrumbs/add-new-item-input.tsx

```diff
@@ -0,0 +1,85 @@
+import { Icon } from '@/powerhouse';
```

Comment:
This will be replaced by a generalized input component that does not have an associated icon